src/Components/ProgressBarStyles.js

```diff
@@ -0,0 +1,65 @@
+const customStepperStyle = {
```

Comment:
Looks like we're no longer using ProgressBar right?

Reply:
I think we are still using it. There was another version in the works that Katie was working on, but when she took a pause, and I took over I prioritized the calculator screens, and felt that the old one was good enough for now. I believe design also has some additional ideas around the progressbar/stepper. I just left the new stuff in so that it can be picked up again when it makes sense.